### PR TITLE
Add restore from backup function

### PR DESCRIPTION
**Pull request by yennchen**

### CODE REVIEW

File: aws/main.tf [hunk 1]
```diff
@@ -26,10 +26,12 @@ data "template_file" "user_data" {
     s3_backup_region         = "${var.s3_backup_region}"
     s3_backup_bucket         = "${var.s3_backup_bucket}"
     s3_backup_schedule       = "${var.s3_backup_schedule}"
+    s3_backup_restore        = "${var.s3_backup_restore}"
     fluentd_image            = "${var.fluentd_image}"
     fluentd_config           = "${var.fluentd_config}"
     grok_pattern             = "${var.grok_pattern}"
     grok_patterns_file       = "${var.grok_patterns_file}"
+    s3cmd_version            = "${var.s3cmd_version}"
```
Comment (coufalja) [inline aws/main.tf:34]: There is awsless already installed wouldn't that suffice? If not could you replace awsless usage with s3cmd and remove it?

**Review by coufalja — CHANGES_REQUESTED** [review]

**Review by r2ronoha — DISMISSED** [review]

File: aws/main.tf (hunk 1 above)
Comment (yennchen) [reply aws/main.tf:34]: awsless doesn't seems to support single file download, it only provides a sync if I'm not wrong. I can replace the awsless create with s3cmd put?

Comment (coufalja) [reply aws/main.tf:34]: I think so yes, the awsless was used because it doesn't require python being installed.

**Review by coufalja — APPROVED** [review]

**Comment by gerricchaplin** [comment]
We should really start using s4cmd instead.

**Pull request merged**